### PR TITLE
Updated visit location to use optional chaining

### PR DESCRIPTION
**Pull request by donaldkibet**
At the moment when visit location data doesn't get the location data it fails. This PR fixes this issue

### CODE REVIEW

**Review by drizzentic — COMMENTED** [review]
LGTM!

File: src/widgets/visit/edit-visit-component.tsx [hunk 1]
```diff
@@ -112,7 +112,7 @@ type EditVisitProps = {
 
 type PatientVisitType = {
   encounters: any[];
-  location: any;
+  location?: any;
```
Comment (FlorianRappl) [inline src/widgets/visit/edit-visit-component.tsx]: Can we type these `any`s? What is the type of the `encounters` array entries? What's the type of `location`? Thanks!

**Review by FlorianRappl — COMMENTED** [review]
Looks good to me. Typing improvements would be helpful.

**Review by FlorianRappl — APPROVED** [review]
LGTM! Thanks for the work :beers:!

**Pull request merged**